concrete_datastore/api/v1/views.py

```diff
@@ -1842,7 +1843,7 @@ def check_date_format(date_type, param_values):
                         False,
                         Response(
                             data={
-                                'message': "Wrong date format, should be '{}' or '{}'".format(
+                                'message': "Wrong date format, should be '{}' or 'yyyy-mm-ddThh:mm:ss' or '{}'".format(
```

Comment:
It should have been better to keep the format for

Reply:
or we can also define the `date_time_format` as 
```python
date_time_format = 'yyyy-mm-ddThh:mm:ss[.xxxxxx]Z'
```

(the `[` `]` to say that it's optional), and juste keen the same message as before, no ?

Reply:
Yes